pt/python_introduction/README.md

```diff
@@ -912,7 +912,7 @@ Hi Rachel!
 
 Parabéns! Você acabou de aprender como escrever funções! :)
 
-## Laços de repetiço
+## Laços de repetição
 
 > Leitores em casa: essa parte está disponível em vídeos (em inglês) [Python Basics: For Loop](https://www.youtube.com/watch?v=aEA6Rc86HF0).
```

Comment:
'Leitores em casa' fica um pouco estranho, não seria melhor algo como 'Leitura em casa' ou outra coisa

Reply:
Isso já estava na tradução, por que você fica comentando coisa que não tem a ver com o PR dela?

This was already in the previous version, why do you keep commenting on stuff that doesn't concern this PR?